README.md

```diff
@@ -179,3 +179,17 @@ Note that the contents of those file will be overidden on each run.
 
 We generate a self-signed wildcard certificate to use for all the ingress
 controllers.
+
+### How to build docker image with the right tag
+
+docker build -t kitesproject/net-test:1.0 /docker/kites/net-tests/
```

Comment:
@simonezacca you need to say from where. And there's an issue with this approach.

If you don't save this to a registry, you need to build the image on each node. I think we should deploy a registry and push the image there, or explain that you don't actually have to build this image, and this is just FYI.